anvil-rstudio-bioconductor/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 3.13.1 - 6/14/2021
```

Comment:
I think this should be 3.13.2? I think we already pushed 3.13.1 for a bugfix.

Reply:
Apologies for the confusion @rtitle . I think it's all straight now.

Reply:
Or maybe I messed it all up.....trying to fix for 3.13.1 and 3.13.2 ...any help is appreciated...or I purge the PR and redo